tests/cloud_test_logic/cloud_test_index.py

```diff
@@ -5,113 +5,117 @@ class CloudTestIndex(str, Enum):
+    structured_text_index_with_custom_model_and_text_preprocessing = "test_index_str_text_custom_model"
     # no_model_index = "test-index-no-model"
 
 
 index_name_to_settings_mappings = {
     CloudTestIndex.unstructured_basic_index: {
         "type": "unstructured",
-        "model": "sentence-transformers/all-MiniLM-L6-v2",
+        "treatUrlsAndPointersAsImages": True,
+        "model": "open_clip/ViT-B-32/laion2b_s34b_b79k",
+
+        "inferenceType": "marqo.GPU",
+        "storageClass": "marqo.performance",
+        "numberOfInferences": 2,
+    },
+    CloudTestIndex.unstructured_image_index_with_image_preprocessing: {
+        "type": "unstructured",
+        "treatUrlsAndPointersAsImages": True,
+        "model": "open_clip/ViT-B-32/laion400m_e32",
+
         "inferenceType": "marqo.CPU.large",
+        "storageClass": "marqo.balanced",
+        "numberOfReplicas": 2,
+    },
+    CloudTestIndex.unstructured_text_index_with_custom_model_and_text_preprocessing: {
+        "type": "unstructured",
+        "treatUrlsAndPointersAsImages": False,
+        "model": "test-model",
+        "modelProperties": {
+            "name": "sentence-transformers/multi-qa-MiniLM-L6-cos-v1",
+            "dimensions": 384,
+            "tokens": 128,
+            "type": "sbert"
+        },
+        "normalizeEmbeddings": True,
+        "textPreprocessing": {
+            "splitLength": 2,
+            "splitOverlap": 1,
+            "splitMethod": "sentence",
+        },
+        "inferenceType": "marqo.CPU.small",
         "storageClass": "marqo.basic"
     },
-    # CloudTestIndex.unstructured_image_index: {
-    #     "type": "unstructured",
-    #     "treatUrlsAndPointersAsImages": True,
-    #     "model": "open_clip/ViT-B-32/laion400m_e32",
-    #     "inferenceType": "marqo.CPU.large",
-    #     "storageClass": "marqo.basic"
-    # },
-    # CloudTestIndex.unstructured_text_index_with_custom_model: {
-    #     "type": "unstructured",
-    #     "treatUrlsAndPointersAsImages": False,
-    #     "model": "test-model",
-    #     "modelProperties": {
-    #         "name": "sentence-transformers/multi-qa-MiniLM-L6-cos-v1",
-    #         "dimensions": 384,
-    #         "tokens": 128,
-    #         "type": "sbert"
-    #     },
-    #     "normalizeEmbeddings": True,
-    #     "textPreprocessing": {
-    #         "splitLength": 2,
-    #         "splitOverlap": 1,
-    #         "splitMethod": "sentence",
-    #     },
-    #     "inferenceType": "marqo.CPU.large",
-    #     "storageClass": "marqo.basic"
-    # },
-    # CloudTestIndex.unstructured_image_index_with_preprocessing_method: {
-    #     "type": "unstructured",
-    #     "treatUrlsAndPointersAsImages": True,
-    #     "model": "open_clip/ViT-B-16/laion400m_e31",
-    #     "imagePreprocessing": {
-    #         "patchMethod": "simple"
-    #     },
-    #     "inferenceType": "marqo.CPU.large",
-    #     "storageClass": "marqo.basic"
-    # },
+
```

Comment:
we need more entropy in the selected fields for structured indexes:
e.g. (add even more entropy if you can think of it!):
- bools
- floats
- ints
- fields used for filtering
- fields used for score modification
- field _not used for filtering_
- fields used for lexical
- text fields _not_ used for lexical
-